src/web/client/dal/remoteFetchProvider.ts

```diff
@@ -345,6 +346,10 @@ async function processDataAndCreateFile(
             let fileCreationValid = true;
             let fileNameWithExtension = GetFileNameWithExtension(entityName, fileName, languageCode, fileExtension);
 
+            const filePath = entityFileMap.get(entityId) ?? [];
+            filePath.push(fileNameWithExtension);
```

Comment:
fileNameWithExtension is just the full file name right? Not the fileUri - check once

Reply:
changing to fileuri